src/components/ChangePlan/ChangePlan.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { useIntl } from 'react-intl';
+import { useRouteMatch } from 'react-router-dom';
+import { Helmet } from 'react-helmet';
+
+const ChangePlan = () => {
+  const { params } = useRouteMatch();
+  const { planId, discountId, promoId } = params;
+  const safeDiscountId = discountId ? discountId : 0;
```

Comment:
Why not add a safePlanId? 🤔

Reply:
The plan is a required parameter, must be there. The other ones are optionals.